sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -421,12 +421,17 @@ mathematicalFunctionName
     : ABS | CBRT | CEIL | CEILING | CONV | CRC32 | E | EXP | EXPM1 | FLOOR | LN | LOG | LOG10 | LOG2 | MOD | PI | POW | POWER
     | RAND | ROUND | SIGN | SQRT | TRUNCATE
     | trigonometricFunctionName
+    | arithmeticFunctionName
```

Comment:
I did this mostly for aesthetic reasons, I noticed that the trigonometric functions were separated and decided to also take out the arithmetic functions since they are also processed in a different file